# HG changeset patch
# Parent bc7bffab1f202be586f9f9f0765cc9cd45ea59e8
-Add: CTRL+ALT+D as global hotkey to toggle text direction

diff --git a/src/main_gui.cpp b/src/main_gui.cpp
--- a/src/main_gui.cpp
+++ b/src/main_gui.cpp
@@ -212,6 +212,7 @@ enum {
 	GHK_CONSOLE,
 	GHK_BOUNDING_BOXES,
 	GHK_DIRTY_BLOCKS,
+	GHK_TEXT_DIRECTION,
 	GHK_CENTER,
 	GHK_CENTER_ZOOM,
 	GHK_RESET_OBJECT_TO_PLACE,
@@ -316,6 +317,10 @@ struct MainWindow : Window
 			case GHK_DIRTY_BLOCKS:
 				ToggleDirtyBlocks();
 				return ES_HANDLED;
+
+			case GHK_TEXT_DIRECTION:
+				ToggleTextDirection();
+				return ES_HANDLED;
 		}
 
 		if (_game_mode == GM_MENU) return ES_NOT_HANDLED;
@@ -473,6 +478,7 @@ static Hotkey global_hotkeys[] = {
 	Hotkey(WKC_BACKQUOTE, "console", GHK_CONSOLE),
 	Hotkey('B' | WKC_CTRL, "bounding_boxes", GHK_BOUNDING_BOXES),
 	Hotkey('I' | WKC_CTRL, "dirty_blocks", GHK_DIRTY_BLOCKS),
+	Hotkey('D' | WKC_CTRL | WKC_ALT, "text_direction", GHK_TEXT_DIRECTION),
 	Hotkey('C', "center", GHK_CENTER),
 	Hotkey('Z', "center_zoom", GHK_CENTER_ZOOM),
 	Hotkey(WKC_ESC, "reset_object_to_place", GHK_RESET_OBJECT_TO_PLACE),
